.github/workflows/ci.yml

```diff
@@ -103,16 +103,31 @@ jobs:
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-
-      - name: Delete Unchanged Files
+        with:
+          fetch-depth: 0
+        
+      - name: Get Changed Files
+        uses: tj-actions/changed-files@1d6e210c970d01a876fbc6155212d068e79ca584
         id: changed-files
-        uses: Pandapip1/delete-unchanged-files@0937c94da2e9f9f262bbf2035e75065a1fd04bcf
+        with:
+          separator: ","
```

Comment:
I assume the hack is because we can't just use `separator: "\n"` for some reason?

Reply:
Yes.